spec/models/manageiq/providers/kubernetes/container_manager/refresher_spec.rb

```diff
@@ -69,6 +69,7 @@ def assert_table_counts
     expect(ContainerImage.count).to eq(3)
     expect(ContainerImageRegistry.count).to eq(1)
     expect(ContainerComponentStatus.count).to eq(3)
+    expect(PersistentVolume.count).to eq(1)
```

Comment:
IIUC the breakage is that `@ems.persistent_volumes.count` is 0, even after first refresh.
I'm not sure we want 3 refreshes (it's a bit heavy test), though this case proves it might be good to have. 
In any case consider adding/changing check for `@ems.persistent_volumes.count`.  I'd even change all checks in this function to test the relations.

Also can you add `assert_specific_persistent_volumes` function?  Worth testing well, given repeated bugs with them...